--max_length 300 --path_to_file ~/rds/hpc-work/plcop/encoder/data/clauses.txt --embedding_dim 1024 --units 256 --batch_size 128 --epochs 100 --checkpoint_freq 5 --checkpoint_dir ~/rds/hpc-work/plcop/encoder/training_checkpoints --checkpoint_prefix ed-1024-u-256
Changed directory to /rds/user/vt289/hpc-work/plcop.

JobID: 37154583
======
Time: Fri  2 Apr 17:17:08 BST 2021
Running on master node: gpu-e-1
Current directory: /rds/user/vt289/hpc-work/plcop

Nodes allocated:
================
gpu-e-1

numtasks=1, numnodes=1, mpi_tasks_per_node=1

Executing command:
==================
python3 ~/rds/hpc-work/plcop/encoder/main.py --max_length 300 --path_to_file ~/rds/hpc-work/plcop/encoder/data/clauses.txt --embedding_dim 1024 --units 256 --batch_size 128 --epochs 100 --checkpoint_freq 5 --checkpoint_dir ~/rds/hpc-work/plcop/encoder/training_checkpoints --checkpoint_prefix ed-1024-u-256

2021-04-02 17:17:11.336486: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library libcudart.so.11.0
Traceback (most recent call last):
  File "/home/vt289/rds/hpc-work/plcop/encoder/main.py", line 256, in <module>
    main()
  File "/home/vt289/rds/hpc-work/plcop/encoder/main.py", line 225, in main
    input_tensor = init_context(prediction_phase=True)
  File "/home/vt289/rds/hpc-work/plcop/encoder/main.py", line 169, in init_context
    tokenizer = utils.load_lang(args)
  File "/rds/user/vt289/hpc-work/plcop/encoder/utils.py", line 99, in load_lang
    json_file = open(lang_file_name(args), 'r')
FileNotFoundError: [Errno 2] No such file or directory: './saved_models/lang.json'
